.github/workflows/ci.yml

```diff
@@ -7,16 +7,18 @@ on:
       - "gh-pages/*"
       - "main"
       - "master"
+  pull_request:
+    types: [opened, reopened]
```

Comment:
I think this can be removed, I think this is a bug in the toolbox resulting the CI workflows to be executed twice for the initial push of an pr.